src/utils/fetch-youtube-data.js

```diff
@@ -0,0 +1,32 @@
+import { buildQueryString } from '../utils/query-string';
+
+const API_ENDPOINT = 'https://www.googleapis.com/youtube/v3/playlistItems';
+const YT_API_KEY = 'Your-Google-API-Key';
```

Comment:
I believe @greggb is still working on helping us get an API key here, but we can add that in afterwards